corehq/apps/reports/util.py

```diff
@@ -1070,3 +1071,10 @@ def get_commcare_version_and_date_from_last_usage(last_submission=None, last_dev
         date_of_use = DateTimeProperty.deserialize(date_of_use)
 
     return version_in_use, date_of_use
+
+
+def make_url(base_url, params):
+    return '{base_url}?{params}'.format(
+        base_url=base_url,
+        params=urlencode(params, True),
+    )
```

Comment:
nit: I like f-strings 
```suggestion
    return f'{base_url}?{urlencode(params, True)}'
```

Reply:
End up using `reverse` in f2c6a12154d8316d3cc13cccf9cbd88275289775